doc/nml.md

```diff
@@ -0,0 +1,965 @@
+# Introduction
```

Comment:
While I am fine with changing over README (the Org renderer on GH is not a nice as I would like), this file is more my personal notes than anything else. Orgmode holds a number of advantages there (for me at least), so I would prefer keeping that. If it ever turns into more that that, fine ;)